src/status_im2/contexts/quo_preview/animated_header_list/animated_header_list.cljs

```diff
@@ -34,16 +34,20 @@
      [quo/text {:size :paragraph-2}
       "Some random description • Developer • Designer • Olympic gold winner • President • Super Hero"]]}])
 
-(defn f-display-picture-comp
+(defn display-picture-comp
   [animation]
-  [reanimated/fast-image
-   {:style (reanimated/apply-animations-to-style
-            {:width  animation
-             :height animation}
-            {:border-radius 72})
-    :source
-    {:uri
-     "https://upload.wikimedia.org/wikipedia/commons/thumb/3/3a/Cat03.jpg/1200px-Cat03.jpg"}}])
+  [:f>
```

Comment:
could you pls elaborate, on this change

Reply:
was just a leftover, sorry, removed it right now